price_list.py

```diff
@@ -65,8 +65,8 @@ def compute(self, product, quantity, uom, pattern=None):
                 product, quantity, product.default_uom)
             if base_price is not None:
                 if line.discount_rate is not None:
```

Comment:
In case has not line.discount_rate but has base_price, can not return base_price (and not pass the test).